appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -434,12 +443,37 @@
                          registeredComponentName eventName)
                        #f))))
 
+       ;; For the HandlesEventDispatching interface
+       (define (dispatchMultiEvent eventName :: java.lang.String
```

Comment:
@puravidaapps and @gitars 

It's also useful even without dynamically created components.  For example, the expression in the "for component" socket doesn't have to just be a constant (e.g. "Button1");  it could be a variable, procedure call, etc.